netkan/netkan/download_counter.py

```diff
@@ -116,12 +211,25 @@ def __init__(self, nk_repo: NetkanRepo, ckm_repo: CkanMetaRepo, github_token: st
         )
 
     def get_counts(self) -> None:
+        graph_query = GraphQLQuery(self.github_token)
         for netkan in self.nk_repo.all_nk_paths():
             netkanDl = NetkanDownloads(netkan, self.github_token)
-            count = netkanDl.get_count()
-            if count > 0:
-                logging.info('Count for %s is %s', netkanDl.identifier, count)
-                self.counts[netkanDl.identifier] = count
+            if netkanDl.kref_src == 'github':
+                # Process GitHub modules together in big batches
+                graph_query.add(netkanDl)
+                if graph_query.full():
+                    # Run the query
+                    graph_query.get_result(self.counts)
+                    # Start over with fresh query
+                    graph_query = GraphQLQuery(self.github_token)
+            else:
```

Comment:
This will work perfectly and likely not a huge deal (being that it's very counter specific), but this logic should be owned by `GraphQLQuery`.